docs/docs/api/promise.monitor.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: api
+id: promise.monitor
+title: Promise.monitor
+---
+
+
+[← Back To API Reference](/docs/api-reference.html)
+<div class="api-code-section"><markdown>
+##Promise.monitor
+
+```js
+Promise.enableMonitoring() -> undefined
```

Comment:
It is done to be compliant to standard in code: longStackTraces can be enabled via envariable, config, and method call, this feature is similar, no?